packages/devtools_app/lib/src/screens/inspector/inspector_data_models.dart

```diff
@@ -57,12 +55,12 @@ const overflowEpsilon = 0.1;
 ///    sum(renderSize) = maxSizeAvailable
 ///
 List<double> computeRenderSizes({
-  @required Iterable<double> sizes,
-  @required double smallestSize,
-  @required double largestSize,
-  @required double smallestRenderSize,
-  @required double largestRenderSize,
-  @required double maxSizeAvailable,
+  required Iterable<double?> sizes,
```

Comment:
can this be non-nullable?

Reply:
Nope. It gets down to LayoutProperties's width and height that are nullable by design.